docs/reference.rst

```diff
@@ -152,6 +152,10 @@ Optional options:
 
   Defaults to ``message``.
 
+``disable_download_auth_header=<boolean>``
+  Whether to send Authorization header for download requests.
+  Defaults to ``false``
+
```

Comment:
The name of the option is confusing. How about `send_download_auth_header` and let it default to `true`?